src/serializer/serializer.js

```diff
@@ -171,6 +171,7 @@ export class Serializer {
         if (onError) {
           onError(realm, res.value);
         }
+        console.log("Execution returned a completion!");
```

Comment:
We are already logging something. Logging it (possibly again) to the console seems uncalled for.